doc/monads/functors.lean

```diff
@@ -0,0 +1,215 @@
+/-!
+# Functor
+
+A `Functor` is any type that can act as a generic container that allows you to transform the
+underlying values inside the container using a function, so that the values are all updated, but the
+structure of the container is the same. This is called "mapping".
+
+A List is the most basic example of a `Functor`.
```

Comment:
add "one of"?

Reply:
fixed in https://github.com/leanprover/lean4/pull/1505/commits/e53237c5a9afcfcfd2536283350598f393081ab3